src/System.Security.Cryptography.Pkcs/ref/System.Security.Cryptography.Pkcs.cs

```diff
@@ -52,8 +52,11 @@ public sealed partial class AlgorithmIdentifier
     }
     public sealed partial class CmsRecipient
     {
+        public CmsRecipient(System.Security.Cryptography.X509Certificates.X509Certificate2 certificate, System.Security.Cryptography.RSAEncryptionPadding rsaEncryptionPadding) { }
+        public CmsRecipient(System.Security.Cryptography.Pkcs.SubjectIdentifierType recipientIdentifierType, System.Security.Cryptography.X509Certificates.X509Certificate2 certificate, System.Security.Cryptography.RSAEncryptionPadding rsaEncryptionPadding) { }
         public CmsRecipient(System.Security.Cryptography.Pkcs.SubjectIdentifierType recipientIdentifierType, System.Security.Cryptography.X509Certificates.X509Certificate2 certificate) { }
         public CmsRecipient(System.Security.Cryptography.X509Certificates.X509Certificate2 certificate) { }
+        public System.Security.Cryptography.RSAEncryptionPadding RSAEncryptionPadding { get { throw null; } }
```

Comment:
These new members probably need to go in the `.netcoreapp` version of the file.  (This is currently building netstandard2.0, and netfx doesn't define these members.)

Reply:
OK, that makes sense. I moved most of the new tests to files that are `netcoreapp` only as well.